Documentation/Contributing/UnitTests.md

```diff
@@ -15,6 +15,8 @@ Before submitting a pull request, make sure to:
 
 1. If writing a feature, write new tests to prevent upcoming code changes breaking this feature.
 
+Currently playmode tests are meant to be run in Unity 2018.4.1f1 and may fail in other versions of Unity
```

Comment:
I'm not sure it's specific to `.1f1`. Our pipelines run them on 2018.4.6 and .12. Were you seeing otherwise?

```suggestion
Currently playmode tests are meant to be run in Unity 2018.4 and may fail in other versions of Unity
```

Reply:
The documentation mentioned .1f1 in a later section so I put it as 1f1 on top for consistency, should we change it to 2018.4 in both areas then?

Reply:
Yeah let's do it!